### ISSUE TITLE
Syrian flag emoji needs urgent Update

### ISSUE THREAD

**Issue by Brazeg** [googlefonts/noto-emoji#495]
The Syrian independence flag (green, white, and black horizontal stripes with three red stars) needs to be implemented as the Syrian flag emoji to reflect current changes.

Flag specifications:
- Three horizontal stripes: green (top), white (middle), black (bottom)
- Three red stars centered on white stripe
- Standard flag proportions

![Image](https://github.com/user-attachments/assets/63f4dfc9-6ec5-482b-aedf-bf1a933cd8ac)

**Comment by jbicha** [comment]
Do you have any references for this being the official flag of Syria other than Wikipedia?

I suggest reading https://blog.emojipedia.org/what-about-the-afghanistan-flag-emoji/ , not for Afghanistan but for Mauritania. It wasn't until [2021](https://emojipedia.org/google/android-12.0/flag-mauritania) that this font was updated for that change.

**Comment by Brazeg** [comment]
The flag change is occurring today (Dec 8, 2024) with Syrian embassies worldwide beginning to change their flags to the independence flag (green-white-black with three red stars).

More official references will be available soon.

The Mauritania example you referenced was a previous flag update that was successfully implemented in Noto emoji (as seen in versions from 2020-2024). I'm following a similar process here to request the Syrian flag update.

**Comment by Naddimou** [comment]
💚💚💚💚💚💚💚💚💚💚💚
🤍🤍🤍❤️🤍❤️🤍❤️🤍🤍🤍
🖤🖤🖤🖤🖤🖤🖤🖤🖤🖤🖤

**Comment by jbicha** [comment]
While this may seem urgent to you, I expect that if this request gets done months from now, it will be one of the fastest emoji flag changes ever. The Mauritania change took years for emoji vendors to implement. I wouldn't expect any action on this from Google this year.

(I am not a Google employee nor a maintainer of this project.)

**Comment by Omaaar90** [comment]
## Context
Recent political developments in Syria have led to the overthrow of President Bashar al-Assad's government. The new leadership has reinstated the "Independence Flag," symbolizing the nation's transition and aspirations for the future. This flag, featuring three horizontal stripes of green, white, and black with three red stars in the white stripe, has been adopted as the official national flag. ([Source: Wikipedia](https://en.wikipedia.org/wiki/Flag_of_Syria))

## Request
We request the update the Syrian flag emoji to reflect the newly adopted national flag design. This update will ensure that digital communications accurately represent Syria's current national symbols.

## Details
- **New Flag Design:**
  - *Description:* The flag consists of three horizontal stripes—green (top), white (middle), and black (bottom)—with three red five-pointed stars centered in the white stripe.
  - *Adoption:* Following the fall of Assad's regime, the "Independence Flag" has been raised within Syria and at Syrian embassies worldwide. ([Source: Reuters](https://www.reuters.com/world/middle-east/syrian-rebel-supporters-hoist-flags-embassies-abroad-2024-12-08/))

- **Geopolitical Context:**
  - *Overthrow of Assad's Government:* Syrian rebels have taken control of major cities, leading to the collapse of the previous regime. ([Source: Reuters](https://www.reuters.com/world/middle-east/syria-rebels-celebrate-captured-homs-set-sights-damascus-2024-12-07/))
  - *Adoption of New Symbols:* The new leadership has introduced national symbols, including the updated flag, to mark a new era. ([Source: Firstpost](https://www.firstpost.com/explainers/syria-rebel-flag-official-bashar-al-assad-fall-13842906.html))

- **Importance:**
  - *Accurate Representation:* Updating the emoji ensures that digital platforms reflect Syria's current national symbols, maintaining cultural and political accuracy.
  - *Public Sentiment:* The new flag holds significant meaning for Syrians and supporters worldwide, symbolizing hope and change.

## References
- [Syrian rebels topple Assad who flees to Russia in Mideast shakeup](https://www.reuters.com/world/middle-east/syria-rebels-celebrate-captured-homs-set-sights-damascus-2024-12-07/)
- [Syrian rebel supporters hoist flags at embassies abroad](https://www.reuters.com/world/middle-east/syrian-rebel-supporters-hoist-flags-embassies-abroad-2024-12-08/)
- [After fall of Assad, will Syria get a new flag?](https://www.firstpost.com/explainers/syria-rebel-flag-official-bashar-al-assad-fall-13842906.html)

## Action Needed
1. **Update the Syrian Flag Emoji:** Modify the existing Syrian flag emoji in the Noto Color Emoji set to reflect the new design.
2. **Coordinate with Platforms:** Ensure that platforms using Noto Color Emoji incorporate the updated flag design in their systems.

Thank you for considering this request. I am available to provide further information or assistance to facilitate this update.

**Comment by jbicha** [comment]
Wikipedia is not an official source for a country changing its flag design.

I also consider some embassies flying this alternate flag design to be more a show of support rather than an official declaration of a flag change.

It is unnecessary to leave further comments of support for this request. For many reasons, Google does not move urgently to change flag designs in their emoji fonts. That does not mean that your request has not been seen nor does it mean that Google will not eventually change the design.

**Comment by HotDog640** [comment]
Just thought to add input. Yes, wikipedia is not an official source. However, the website of the Prime Minister of Syria (pministry.gov.sy) shows the new flag within an updated coat of arms in the website's navbar. Additionally, the site has an image of a recent Council of Ministers session inside the Syrian Parliament showcasing the flag used in an official capacity. The parliament's Facebook page (their website being offline) has also began to use the new flag. These are official channels using the new flag.

**Comment by sirtb** [comment]
Syria's new constitution has officially declared this new flag as the official one

**Comment by AbdullahMRiad** [comment]
Apple and WhatsApp have already implemented the new Syrian flag

[emojipedia page for reference](https://emojipedia.org/flag-syria#designs)

**Comment by sirtb** [comment]
OneUI and Openmoji have also implemented the new Syrian flag

**Comment by jagomf** [comment]
Trump is meeting with the Syrian president right now, you will probably find hundreds of pictures with the new official flag all around @jbicha

**Comment by berylline** [comment]
Since Google doesn't accept PRs for Noto Color Emoji, at least when it comes to changing flag emojis themselves, is there a process and/or a timeline for when the Syrian flag emoji gets updated?

**Comment by SaleemMaini** [comment]
please this update is needed for all of us 
there isn't any reason to not produce it.

**Comment by Bilal2453** [comment]
I believe it is probably worth giving this issue the proper labels now that things got more clear? can @jbicha please help with that? would also be appreciated if it got assigned as a goal for *some* release number (internally at least!). Thanks.

**Comment by ChiefMikeK** [comment]
`We don't accept pull requests, or design requests in general.`  
👎🏽 [Google has Updated this CONTRIBUTING.md](https://github.com/googlefonts/noto-emoji/blob/main/CONTRIBUTING.md)
* 😓

**Comment by Bilal2453** [comment]
> `We don't accept pull requests, or design requests in general.`  
> 👎🏽 [Google has Updated this CONTRIBUTING.md](https://github.com/googlefonts/noto-emoji/blob/main/CONTRIBUTING.md)
> * 😓
> 

Yeah I've noticed this (on other Google repos as well), although this issue is neither a PR nor a design request per se, and whether it's accepted or not the font has to update this emoji eventually, internally it needs to be tracked so might as well be transparent about the process.
